src/CodeGen_C.cpp

```diff
@@ -2550,13 +2562,24 @@ void CodeGen_C::visit(const Call *op) {
             rhs << "(" << print_type(op->type) << ")";
         }
         rhs << "(&" << struct_name << ((count > 0) ? "[0])" : ")");
-    } else if (op->is_intrinsic(Call::load_struct_member)) {
+    } else if (op->is_intrinsic(Call::load_typed_struct_member)) {
+        // Given an instance of a typed_struct, a definition of a typed_struct,
+        // and the index of a slot, load the value of that slot.
+        //
+        // An instance of a typed_struct is an Expr of Handle() type that has been
+        // created by a call to make_typed_struct.
+        //
+        // An definiton of a typed_struct is an Expr of Handle() type that has been
```

Comment:
change "An" to "A".

It is probably worth covering that both are needed because the instance is often a void * by the time it is being used. E.g. it has been passed through an API that takes an opaque pointer as a void *. Thus it needs to be casted back to the correct type so its fields can be accessed.

Reply:
Done